core/common/src/main/java/alluxio/util/ShellUtils.java

```diff
@@ -135,8 +136,8 @@ public static UnixMountInfo parseMountInfo(String line) {
   }
 
   @NotThreadSafe
-  private static final class Command {
-    private String[] mCommand;
+  private static class Command {
```

Comment:
Removed `final` to be inherited by SshCommand class. I don't see much benefit from stopping this class being inherited.

Reply:
does it make sense to make `Command` a real class with its own file?

Reply:
Done